config/setupTests.js

```diff
@@ -3,8 +3,13 @@ import 'whatwg-fetch';
 import 'babel-polyfill';
 import '@testing-library/jest-dom';
 
+import Adapter from '@cfaester/enzyme-adapter-react-18';
+import { configure } from 'enzyme';
```

Comment:
We don't want enzyme for testing. It's not maintained anymore. We are using the `@testing-library/react` for unit testing.